src/main/java/org/candlepin/model/ExporterMetadata.java

```diff
@@ -56,7 +56,7 @@ public class ExporterMetadata extends AbstractHibernateObject {
 
     @Column(nullable = false)
     @NotNull
-    private Date exported;
+    private OffsetDateTime exported;
```

Comment:
ok, then. I'll take care of it

Reply:
OpenAPI can use whatever it wants -- OffsetDateTime is somewhat fine (it's basically just an instant + offset). However internally we aren't interested in storing all those random offsets, and we only work in terms of UTC, so we should only be storing instants.

The conversion between the two is easy enough -- `OffsetDateTime.toInstant()` gives us just the value we need which can then be stored and compared safely.

Reply:
We could use either OffsetDateTime or Instant on the API. If we used Instant it would mean we'd have to change our custom serializer/deserializers for OffsetDateTime to support Instant instead. Either way, we should be able to accept as input (and serialize as output) ISO 8601 date strings with and without offset, and parse it into whatever class we want. So we don't gain much by switching to Instant except not having to convert between the two in the DTO-model boundary.